flybywiresim/a32nx
# fix(apu): hz above then below normal during start

Fixes #5317

## Summary of Changes
Fixes a situation in which the APU GEN's frequency briefly goes above and then below the normal operational range at which it can provide electricity (390-410 Hz), thus causing various buses to become powered, then unpowered and then powered again; which in turn would cause various sounds associated with bus power state to play twice.

The window of opportunity for this to occur wasn't very large, and thus most of the time users wouldn't run into this or notice it. Solved by calculating and programming another polynomial curve. Includes a unit test to capture any future regression.

## Additional context



Discord username (if different from GitHub): SjotgunSjonnie#9623

## Testing instructions
The window of opportunity for this to occur wasn't very large, and thus most of the time users wouldn't run into this or notice it. 

Honestly, I don't think this needs testing as the unit tests captures this already 😉. However, if you must, then:

1. Start the APU without EXT PWR being ON.
2. Observe the `A32NX_ELEC_AC_1_BUS_IS_POWERED` local variable doesn't go from 0 to 1 to 0 to 1, but instead from 0 to 1 and that's it.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
